lib/sidekiq/throttler/version.rb

```diff
@@ -1,5 +1,5 @@
 module Sidekiq
   class Throttler
-    VERSION = '0.4.1'
+    VERSION = '0.5.0'
```

Comment:
Please remove this change from your commit. I'll look at what's changed since the last release then tag and push a new version after merging your changes.